source/tutorial/kotlin.txt

```diff
@@ -39,6 +39,8 @@ around 30 minutes.
 
 .. include:: /includes/kotlin-install-prerequisites.rst
```

Comment:
Not sure why this is a separate include file - this is the only spot we use it and it obscures the "Prerequisites" heading.

Mind getting rid of the include and inlining the content here?